x/incentive/keeper/rewards.go

```diff
@@ -521,6 +523,7 @@ func (k Keeper) SynchronizeHardDelegatorRewards(ctx sdk.Context, delegator sdk.A
 	}
 
 	// Add rewards to delegator's hard claim
+	// TODO: types.HardLiquidityRewardDenom
```

Comment:
merge artifact I think- can delete

Reply:
Removed in https://github.com/Kava-Labs/kava/pull/788/commits/8d1cc4a9f8a912b2cfec44b1803c3aaa93c6ff57